bin/citgm.js

```diff
@@ -70,40 +75,42 @@ if (!citgm.windows) {
 
 app.parse(process.argv);
 
+var config = app.config ? util._extend(require(resolve(process.cwd(), app.config)), app) : app;
+
 var log = logger({
-  level:app.verbose,
-  nocolor: !app.color
+  level:config.verbose,
+  nocolor: !config.color
 });
 
 update(log);
 
-if (!app.su) {
+if (!config.su) {
   require('root-check')(); // silently downgrade if running as root...
                            // unless --su is passed
 } else {
   log.warn('root', 'Running as root! Use caution!');
 }
 
 if (!mod) {
-  app.outputHelp();
+  config.outputHelp();
```

Comment:
On second thought (third really), because of the way `commander` works, this approach is still the best, I think. However, `config.outputHelp()` is invalid because the `config` object doesn't have the same prototype as `app`, so this should just be called on `app`. Fix is incoming.